Sources/Scout/Definitions/PathExplorer.swift

```diff
@@ -358,6 +358,15 @@ where
     */
     mutating func add<Type: KeyAllowedType>(_ newValue: Any, at path: PathElementRepresentable..., as type: KeyType<Type>) throws
 
+    // MARK: - Paths
+
+    /// Returns all the path leading to single or group values
+    /// - Parameters:
+    ///   - initialPath: Scope the return paths with this path as a starting point
+    ///   - filter: Optionnaly provide a filter on the key
```

Comment:
Small typos:
```suggestion
    ///   - filter: Optionally provide a filter on the key
```

Reply:
Fixed in latest commit 739ca313022c667ae53fc96f79a81c50d6a8264f